makefile

```
@@ -69,4 +69,20 @@ stack-up-monitoring:
+	cd deployment/docker-compose && docker compose -f docker-compose-local.yaml up
+
+stack-down-local:
+	cd deployment/docker-compose && docker compose -f docker-compose-local.yaml down
+
+stack-up-minio:
+	cd deployment/docker-compose && docker compose -f docker-compose-minio.yaml up
+
+stack-down-minio:
+	cd deployment/docker-compose && docker compose -f docker-compose-minio.yaml down
+
 reup: stack-down build-all stack-up
```

checkmake (0.2.2)

[warning] Target "reup" should be declared PHONY.

(phonydeclared)